### PR TITLE
Replaced webbertakken refs with game-ci and run tests in parallel.

### PR DESCRIPTION
**Pull request by yanniboi**
#### Changes

- Replaced references to `webbertakken` with game-ci
- Run test all modes in parallel as described here: https://github.com/webbertakken/unity-reference-project/blob/main/.github/workflows/main.yml

#### Checklist

- [x] Read the contribution [guide](../CONTRIBUTING.md) and accept the [code](../CODE_OF_CONDUCT.md) of conduct
- [x] Readme (updated or not needed)
- [x] Tests (added, updated or not needed)

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -217,16 +223,17 @@ jobs:
...
       # Configure test runner
-      - uses: webbertakken/unity-test-runner@v2
+      - uses: game-ci/unity-test-runner@v2.0-alpha-5
         id: testRunner
         with:
-          projectPath: test-projects/test-project
-          testMode: all
+          testMode: ${{ matrix.testMode }}
+          checkName: ${{ matrix.testMode }} test results
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
```
Comment (yanniboi) [inline .github/workflows/main.yml:232]: I don't know if we want to go into github tokens here. As I understand it the check doesnt work without them?

Comment (webbertakken) [reply .github/workflows/main.yml:232]: Yes, providing the token allows for the github checks api to be used.

**Comment by webbertakken** [comment]
Excellent!

**Comment by webbertakken** [comment]
The error about the version is not related to this PR per se. See https://github.com/game-ci/unity-test-runner/issues/131

**Comment by webbertakken** [comment]
Actually it's because you removed `projectPath: test-projects/test-project`, so the project can not autodetect the unity editor version.

**Comment by webbertakken** [comment]
That error is because the key is not accessible for PRs from outside the repo, but I saw that it worked correctly now.

**Pull request merged**